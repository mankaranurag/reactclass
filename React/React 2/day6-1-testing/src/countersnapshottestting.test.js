import React from "react";
import Counter from "./countertotest";

import renderer from "react-test-renderer";

describe("Use Snapshot testing", () => {
  it("should take the snapshot", () => {
    var tree = renderer.create(<Counter />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
