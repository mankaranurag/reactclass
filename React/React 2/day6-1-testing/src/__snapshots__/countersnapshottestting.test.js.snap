// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Use Snapshot testing should take the snapshot 1`] = `
<div>
  <p>
    Count : 
    0
  </p>
  <button
    id="incre"
    onClick={[Function]}
  >
    +++
  </button>
</div>
`;
